packages/electron-updater/src/BaseUpdater.ts

```diff
@@ -56,11 +58,23 @@ export abstract class BaseUpdater extends AppUpdater {
 
     try {
       this._logger.info(`Install: isSilent: ${isSilent}, isForceRunAfter: ${isForceRunAfter}`)
```

Comment:
Would you mind moving this below the try-catch and add to the logging sentence `installPathRequiresElevation`, It'd help with future debugging

Reply:
@mmaietta Thank you very much for such a quick reply and for your suggestion, already done:) sorry for the delay